Sources/TuistSupport/Extensions/AbsolutePath+Extras.swift

```diff
@@ -4,6 +4,10 @@ import TSCBasic
 
 let systemGlob = Darwin.glob
 
+enum GlobError: Error {
```

Comment:
We can adopt `FatalError` and add a description, in the event of failure Tuist will print a user friendly error message :)

We could even go a step further and add the the path as an associated value which we can include in the error description.

Here's an example where a similar pattern is followed:

https://github.com/tuist/tuist/blob/master/Sources/TuistLoader/Loaders/ManifestLoader.swift#L8